frontend/src/screens/periods/list.vue

```diff
@@ -8,14 +8,20 @@
       </router-link>
     </div>
     <loader :loading='isLoading'/>
-    <simple-table
+    <simple-table v-if="periods.length"
       :headers="headers"
       :data="periods"
       :actions="{edit: true, delete: true, showViewButton: false}"
       :empty-state-msg="$t('noPeriodsMessage')" 
       @onEdit="onEdit"
       @onDelete="onDelete">
     </simple-table>
+    <div v-if="periods.length == 0">
+      <missing-data-empty-state
```

Comment:
No hace falta el div
```suggestion
      <missing-data-empty-state v-if="!periods.length"
```